lib/chef/http/authenticator.rb

```diff
@@ -16,16 +16,23 @@
 # limitations under the License.
 #
 
+require "chef/mixin/powershell_exec"
+require "chef/mixin/shell_out"
 require_relative "auth_credentials"
 require_relative "../exceptions"
 autoload :OpenSSL, "openssl"
 
 class Chef
   class HTTP
     class Authenticator
-
       DEFAULT_SERVER_API_VERSION = "2".freeze
 
+      extend Chef::Mixin::PowershellExec
+      include Chef::Mixin::PowershellExec
```

Comment:
you shouldn't need to include here since you're only using `powershell_exec` from within class methods, so only need to extend.